src/components/CurriculumComponents/UnitsTabMobile/UnitsTabMobile.tsx

```diff
@@ -0,0 +1,256 @@
+import React, { FC, useState, useRef, useEffect } from "react";
+import { OakP, OakHeading, OakFlex } from "@oaknational/oak-components";
+
+import { Thread } from "../CurriculumVisualiser/CurriculumVisualiser";
+
+import Box from "@/components/SharedComponents/Box";
+import Button from "@/components/SharedComponents/Button/Button";
+import ButtonGroup from "@/components/SharedComponents/ButtonGroup";
+import Radio from "@/components/SharedComponents/RadioButtons/Radio";
+import RadioGroup from "@/components/SharedComponents/RadioButtons/RadioGroup";
+
+// Types and interfaces
+
+type UnitsTabMobileProps = {
+  selectedThread: Thread | null;
+  handleSelectThread: (thread: string) => void;
```

Comment:
I'd rename this param `threadName` / `threadSlug` or whatever it's really expecting to avoid confusion